packages/uicore/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -250,6 +250,11 @@ export function MultiSelect(props: MultiSelectProps): React.ReactElement {
       items={loading ? [{ label: 'Loading...', value: Loading }] : items}
       selectedItems={value}
       onItemSelect={handleItemSelect}
+      onItemsPaste={items => {
+        if (typeof onChange === 'function' && allowCommaSeparatedList) {
```

Comment:
why the if condition on `allowCommaSeparatedList`?
pasting single values should also be allowed 🤔